### PR TITLE
feat(icons): added new icons

### PR DESCRIPTION
**Pull request by danlundgren1**
Added new icons

### Checklist before requesting a review

- [X] I have performed a self-review of my own code
- [ X ] I have verified that the code builds perfectly fine on my local system
- [ / ] I have added tests that prove my fix is effective or that my feature works
- [ / ] I have commented my code, particularly in hard-to-understand areas
- [ X ] I have verified that my code follows the style already available in the repository
- [ / ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by lekoaf** [comment]
@danlundgren1, don't add internal links on an external package...

**Comment by danlundgren1** [comment]
> @danlundgren1, don't add internal links on an external package...

Fixed

**Comment by boilund** [comment]
We use conventional commit so please edit your commit message to `feat(icons): added ICON_NAME icon`.

File: packages/icons/src/copy-outline.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!-- 
+  Based on Material design icons by Google - https://github.com/google/material-design-icons
+  which are licensed under the Apache License 2.0. Icon 'content_copy (outline)'.
+
+  File is modified. Modifications are licensed under the MIT license.
+-->
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm3 4H8c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h11c1.1 0 2-.9 2-2V7c0-1.1-.9-2-2-2zm0 16H8V7h11v14z"/></svg>
```
Comment (boilund) [inline packages/icons/src/copy-outline.svg:7]: All svg have multiple paths or other tags other than path. Please make sure the svg has only one path between svg tag. The guideline is on `DEVELOPMENT.md`.

Comment (danlundgren1) [reply packages/icons/src/copy-outline.svg]: Fixed

**Review by boilund — APPROVED** [review]
LGTM

**Pull request merged**